Assets/Plugins/Source/Editor/EditorWindows/InstallEOSZipWindow.cs

```diff
@@ -249,7 +249,7 @@ protected override void RenderWindow()
                             // This should be the correct directory
                             var projectDir = FileUtility.GetProjectPath();
                             // TODO: Async not tested here.
-                            PackageFileUtility.CopyFilesToDirectory(projectDir, fileResults).Wait();
+                            PackageFileUtility.CopyFilesToDirectory(projectDir, fileResults);
```

Comment:
Changing this line causes a warning to be generated in the Unity Editor. This can be fixed by either discarding the return value like this:
```csharp
_ = PackageFileUtility.CopyFilesToDirectory(projectDir, fileResults);
```
or by adding `async` to the function signature, and changing this line to this:
```csharp
await PackageFileUtility.CopyFilesToDirectory(projectDir, fileResults);
```

Reply:
This has been fixed, so I am marking the conversation as resolved.